### PR TITLE
Add build target on docker plugin

### PR DESCRIPTION
**Pull request by mavimo**
To be merged after https://github.com/drone-plugins/drone-docker/pull/168

### CODE REVIEW

File: content/drone-plugins/drone-docker/index.md [hunk 1]
```diff
@@ -122,6 +122,17 @@ pipeline:
     secrets: [ docker_username, docker_password ]
 ```
 
+Example configuration that allow build a specific target:
```
Comment (tonglil) [inline content/drone-plugins/drone-docker/index.md]: Can you elaborate a bit more on this example? Just saying "target" may not be very clear to new users.

Comment (mavimo) [reply content/drone-plugins/drone-docker/index.md]: @tonglil thx!
Do you think that add a link to https://docs.docker.com/develop/develop-images/multistage-build/#name-your-build-stages should be fine or I should include more elaborate docs about multistage build on docker?

Comment (tonglil) [reply content/drone-plugins/drone-docker/index.md]: Yes, giving some context would be good (link and/or otherwise).

**Comment by tboerger** [comment]
@mavimo if you add the requested changes we can merge this pr.

**Comment by mavimo** [comment]
@tboerger sorry for delay, I'll do it in few mins.

**Comment by mavimo** [comment]
@tboerger @tonglil done, let me know if some more info should be added or need some fix.

**Pull request merged**